### PR TITLE
Installing github actions tool- Pytest

### LINKED ISSUE #57: Installing Pytest
Installing Pytest files as part of Github actions tool

### PR DESCRIPTION
**Pull request by noaramon**
This PR contains Pytest files as part of Github actions tool.
Adding pytest will assist in running automated tests when uploading to github.
(The flake8 file was misspelled so I used this chance to rename it)

Fixes #57

### CODE REVIEW

**Review by ravidbro — CHANGES_REQUESTED** [review]
Remove the old flake configuration that has typo,
You are now running both of them.

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ container: fedora:32
+ steps:
+ - name: Install Pipenv and Git
+ run: dnf install -y pipenv git
+ - name: Checkout code
+ uses: actions/checkout@v2
+ - name: Setup environment
+ run: pipenv sync --dev
+ - name: Run PyTest
+ run: pipenv run pytest -v
```
Comment (ravidbro) [inline .github/workflows/pytest.yml:16]: Add a newline at the end of the file.

File: pytest.ini [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[pytest]
+JANGO_SETTINGS_MODULE = beyond_tutorial.settings
+python_files = tests.py test_*.py *_tests.py
```
Comment (ravidbro) [inline pytest.ini:3]: Add a newline at the end of the file.

**Review by ravidbro — CHANGES_REQUESTED** [review]
I don't see on the PR that the pytest is running

**Comment by noaramon** [comment]
> Remove the old flake configuration that has typo,
> You are now running both of them.

I used git reset --soft HEAD^ and then unstaged the flake8 file but now I got an error that the tip of my current branch is behind. I tried to pull origin Installing-Pytest but it's not solving the problem.

**Comment by ravidbro** [comment]
I meant that the repo already have a file for flake but with typo in the name, so if you add a file with the right name, you need to delete the old file.

**Pull request closed without merging**